admin_pages/about/templates/credits.template.php

```diff
@@ -89,8 +89,8 @@
             'For every major release we want to recognize the people who contributed to the release via a GitHub pull request. Want to see your name listed here? %sWhen you submit a pull request that gets included in a major release%s, we\'ll add your name here linked to your GitHub profile.',
             'event_espresso'
         ),
-        '<a href="https://github.com/eventespresso/event-espresso-core" title="Contribute to Event Espresso by making a pull request via GitHub">',
-        '</a>'
+                            '<a href="https://github.com/eventespresso/event-espresso-core" title="Contribute to Event Espresso by making a pull request via GitHub">',
+                            '</a>'
```

Comment:
this was flagged as a formatting error by PHPCS (CodeSniffer) and fixed automatically by PHPCBF 
# 🤷‍♂️